tests/hazmat/bindings/test_openssl.py

```diff
@@ -112,3 +112,16 @@ def test_rust_internal_error(self):
         assert error.reason == b.lib.EVP_R_DATA_NOT_MULTIPLE_OF_BLOCK_LENGTH
         if not b.lib.CRYPTOGRAPHY_IS_BORINGSSL:
             assert b"data not multiple of block length" in error.reason_text
+
+    def test_ssl_session_reused_new(self):
```

Comment:
No need for a test, we don't add them for all bindings.

Reply:
Is this a request to remove the test?

I think that a test can help with any accedental removal of the API.

It can also serve as an example on how to use the API.

Reply:
Yes, please remove the test.